gm_dotnet_managed/GmodNET/GameConsoleWriter.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+public class GameConsoleWriter : TextWriter
+{
+    private class Tier0
+    {
+        [DllImport("tier0", CallingConvention = CallingConvention.Cdecl)]
+        public static extern void Msg([MarshalAs(UnmanagedType.LPStr)] string msg);
+    }
+
+    private static GameConsoleWriter instance;
+    private uint references;
+
+    public override string NewLine { get => "\n"; }
+    public override Encoding Encoding => throw new NotImplementedException();
```

Comment:
Why are we throwing an exception here?

Reply:
@GlebChili it's code generated by ide, for implementing TextWriter interface.
I'll probably change it to `ASCII` or `UTF8`

Reply:
Yes, we need it to return proper encoding, used by Tier0 Msg

Reply:
resolved in https://github.com/GmodNET/GmodDotNet/pull/70/commits/d2f865e987f0302303e76a918edc08efcb55f5ae